client/my-sites/domains/domain-management/add-google-apps/add-email-addresses-card.jsx

```diff
@@ -288,7 +290,7 @@ const AddEmailAddressesCard = createReactClass( {
 	},
 } );
 
-function getGoogleAppsCartItems( { domains, fieldsets } ) {
+function getGoogleAppsCartItems( { domains, fieldsets, isBusiness } ) {
```

Comment:
It seems like GSuite offers [Basic, Business and Enterprise plans](https://support.google.com/a/answer/1247360?hl=en).

We have a lot of booleans in this patch like "isBusiness". Instead we should probably allow multiple SKUs in case we want to add support for Enterprise in the future.